Dockerfile

```diff
@@ -3,11 +3,11 @@ FROM node:10
 WORKDIR /usr/src/app
 
 # why the *
-COPY package*.json /usr/src/
+COPY package.json .
 RUN npm install
 
 COPY . .
 
 COPY wait-for-it.sh /usr/wait-for-it.sh
-RUN chmod +x /usr/wait-for-it.sh
+# RUN chmod +x /usr/wait-for-it.sh
```

Comment:
Hmmm. I should try commenting this out. I just read that this was necessary in order to run the bash command.

Reply:
Same, it's to make it executable. However, by addressing bash -c that could be diff?